web/index.ts

```diff
@@ -380,7 +370,8 @@ const App = (_: any, state: AppState, setState: SetState) => {
                             const nextImage = images.length === 1
                                 ? 'https://cdn.jsdelivr.net/gh/remojansen/logo.ts@master/ts.svg'
                                 : '';
-                            setLoadingState({ images: [...images, nextImage] })
+
+                            setLoadingState({ images: [...images, nextImage],heights: [...heights,''],widths: [...widths,''] })
```

Comment:
```suggestion
                            setLoadingState({ images: [...images, nextImage], heights: [...heights,''], widths: [...widths,''] })
```

Reply:
Are you guys using prettier or linters by any chance? Is it okay if we add it to the repo ?

Reply:
We have a style-guide that is going to become open source soon so we'll add it to this repo once its available